packages/api-explorer/src/JsonForm/anyOf-custom-editor.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-underscore-dangle */
+const baseCustomEditor = require('./get-custom-editor')
+const Choices = require('choices.js')
+require('choices.js/public/assets/styles/choices.min.css')
+
+module.exports = (setSwitcher) => baseCustomEditor('multiple').extend({
+  build() {
+    const response = this._super()
+    const { switcher, switcherDiv }= this.buildSwitcher()
+    this.switcher.replaceWith(switcherDiv)
+    this.addErrorMessageHtmlNode()
+    switcher.dispatchEvent(new Event('change'))
+    this.showErrorMessage('start-config')
+    setSwitcher(switcher)
+    return response
+  },
+  buildSwitcher() {
+    const self = this
+    const switcher = this.theme.getSwitcher(this.display_text)
+    switcher.multiple = true
+    switcher.selectedIndex = '-1'
+    
+    const switcherDiv = document.createElement('div')
+    const switcherLabel = document.createElement('label')
+    switcherLabel.style.fontWeight = 'bold'
+    switcherLabel.innerText = 'Select the schemas:'
+    switcherDiv.appendChild(switcherLabel)
+    switcherDiv.appendChild(switcher)
+    const choices = new Choices(switcher, {
```

Comment:
Could we use antd select? e.g. using `ReactDOM.render(switcherDiv, antSelect)`

Reply:
There's a problem with using React, since the switcher is added to the DOM by JSONEditor we have no control (at this point of execution) to the element instance in DOM for using react render function and thus get the following error: `Target container is not a DOM element.`